### PR TITLE
D3D11: Expand 16-bit textures to 32-bit if Win8+-only texture formats are not supported

### PR DESCRIPTION
**Pull request by hrydgard**
This should help compatibility of the D3D11 backend with Windows 7.

### CODE REVIEW

**Comment by ufdada** [comment]
This seems to break texts in loco roco 2 on Windows 10

ppsspp-v1.3-705-gb94ad12-windows-amd64:
uces01059_00004

ppsspp-v1.3-709-g364b7e2-windows-amd64:
uces01059_00003

**Comment by hrydgard** [comment]
Huh, I'll take a look.

**Comment by hrydgard** [comment]
I have the game and it works perfectly fine here...

**Comment by ufdada** [comment]
Weird, maybe it broke because i use the original psp fonts? Also i am using the European version (UCES01059).

It's also very interesting they sometimes seem to work properly... Like when they are generated a second time.

**Comment by hrydgard** [comment]
Hm, interesting...

**Comment by ufdada** [comment]
Ok while testing around i found the cause of the problem: it's the texture scaling which is broken. If i turn it off it works just fine

File: GPU/Directx9/GPU_DX9.cpp [hunk 1]
```diff
@@ -473,11 +473,12 @@ void GPU_DX9::UpdateCmdInfo() {
 void GPU_DX9::CheckGPUFeatures() {
 	u32 features = 0;
 
-	features |= GPU_SUPPORTS_4BIT_FORMAT;
+	features |= GPU_SUPPORTS_16BIT_FORMATS;
 	features |= GPU_SUPPORTS_BLEND_MINMAX;
 	features |= GPU_SUPPORTS_TEXTURE_LOD_CONTROL;
 	features |= GPU_PREFER_CPU_DOWNLOAD;
 	features |= GPU_SUPPORTS_ACCURATE_DEPTH;
+	features |= GPU_SUPPORTS_16BIT_FORMATS;
```
Comment (ufdada) [inline GPU/Directx9/GPU_DX9.cpp]: Is this duplication intended?

Comment (hrydgard) [inline GPU/Directx9/GPU_DX9.cpp]: Nope, thanks.

**Comment by hrydgard** [comment]
@ufdada Ah! That makes sense, and easy for me to test. I'll get it fixed soon.

**Comment by ufdada** [comment]
Glad i could help you :)

**Pull request merged**